src/controllers/timeEntryController.js

```diff
@@ -1,6 +1,6 @@
 const moment = require('moment-timezone');
 const mongoose = require('mongoose');
-const { getInfringmentEmailBody } = require('../helpers/userhelper')();
+const { getinfringementEmailBody } = require('../helpers/userhelper')();
```

Comment:
Please change to camelCase in the following files.
[userhelper.js](https://github.com/OneCommunityGlobal/HGNRest/pull/220/files#diff-be4faba0a35ebaba61c8dfaee9d3b275b61ab7b93fa463f23c813a893ddbd1ff)
[userProfileController.js](https://github.com/OneCommunityGlobal/HGNRest/pull/220/files#diff-b1e237c24ca4ab88efa9d26a769fbf9cac04284ecab643bb65f20a2196b7f8ce)
[timeEntryController.js](https://github.com/OneCommunityGlobal/HGNRest/pull/220/files#diff-364400a53fcc145234e7c6ec363c062ec22f4791168e265fdf8df3c75aceb897)

Reply:
I changed some variables to camel case. I did not change all of them because some depend on other files. I will fix these lint problems in another pull request. I was assigned to fix all lint problems.

Reply:
Changes all good! Are you going to fix the rest of this once you fix the lint problems?